pandas/tests/frame/methods/test_quantile.py

```diff
@@ -650,3 +651,61 @@ def test_quantile_ea_scalar(self, index, frame_or_series):
             assert result == expected
         else:
             tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "dtype, expected_data, expected_index, axis",
+        [
+            ["float64", [], [], 1],
+            ["int64", [], [], 1],
+            ["float64", [np.nan, np.nan], ["a", "b"], 0],
+            ["int64", [np.nan, np.nan], ["a", "b"], 0],
+        ],
+    )
+    def test_empty_numeric(self, dtype, expected_data, expected_index, axis):
+        # GH 14564
+        df = DataFrame(columns=["a", "b"], dtype=dtype)
+        result = df.quantile(0.5, axis=axis)
+        expected = Series(
+            expected_data, name=0.5, index=Index(expected_index), dtype="float64"
+        )
+        tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "dtype, expected_data, expected_index, axis, expected_dtype",
+        [
+            ["datetime64[ns]", [], [], 1, "float64"],
```

Comment:
why is this expected float64?

Reply:
Good point; I think this should be `datetime64[ns]`. Opened up https://github.com/pandas-dev/pandas/issues/41544 related to this bug.